go.mod

```diff
@@ -3,25 +3,25 @@ module github.com/cncf-tags/green-reviews-tooling
 go 1.23.2
 
 require (
-	github.com/99designs/gqlgen v0.17.63
-	github.com/Khan/genqlient v0.7.0
-	github.com/vektah/gqlparser/v2 v2.5.21
-	go.opentelemetry.io/otel v1.32.0
+	github.com/99designs/gqlgen v0.17.73
+	github.com/Khan/genqlient v0.8.1
+	github.com/vektah/gqlparser/v2 v2.5.27
+	go.opentelemetry.io/otel v1.34.0
```

Comment:
I upgraded the dagger k3s module to the latest which also updates the go dependencies.

```
dagger install github.com/marcosnils/daggerverse/k3s@v0.1.10
```

See https://daggerverse.dev/mod/github.com/marcosnils/daggerverse/k3s@28eea1fcf3b6ecb38a628186107760acd717442f